src/widgets/editor.py

```diff
@@ -78,35 +96,62 @@ def updatePreferences(self,*args):
     
     def new(self):
         
-        self._filename = ''
+        self.update_filename('')
```

Comment:
Maybe a nice use case for a python @property

Reply:
Done.